serverless.yml

```diff
@@ -202,6 +202,12 @@ resources:
       Properties:
         FifoQueue: true
         QueueName: outbound-sms-dlq-${self:provider.stage}.fifo
+
+    # SYSTEM TEST
+    SystemTestQueue:
+      Type: AWS::SQS::Queue
+      Properties:
+        QueueName: system-test-${self:provider.stage}
```

Comment:
I'd set `MessageRetentionPeriod` to 60 (one minute, the minimum) so that data from prior failed runs is less likely to mess up subsequent tests.